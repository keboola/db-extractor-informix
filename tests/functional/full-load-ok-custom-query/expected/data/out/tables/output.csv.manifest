{
    "destination": "output",
    "incremental": false,
    "primary_key": [
        "id"
    ]
}
