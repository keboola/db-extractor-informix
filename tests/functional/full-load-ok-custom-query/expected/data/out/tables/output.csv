"name","date","id"
"Jack Dawson","2020-10-03 01:02:34.000","1"
"Xander Thomas","","2"
"Jay Macdonald","2020-10-01 10:20:30.000","3"
